src/data/missions.csv

```diff
@@ -1294,3 +1294,4 @@
 "【デイリーミッション】\nレジェンドレースに1回出走しよう","[Daily] Participate the legend race once"
 "【デイリーミッション】\nレジェンドレースに2回出走しよう","[Daily] Participate the legend race 2 times"
 "【デイリーミッション】\nレジェンドレースに3回出走しよう","[Daily] Participate the legend race 3 times"
+"【デイリーミッション】\nタウラス杯にエントリーしよう","[Daily] Make an entry in the Taurus Hai"
```

Comment:
"Enter the Taurus Hai" sounds better imo.

Reply:
The original one its Taurus Cup right?

Reply:
What do you mean?